extensions/bitwarden/src/api.ts

```diff
@@ -69,7 +111,12 @@ export class Bitwarden {
     return stdout;
   }
 
-  private async exec(args: string[], abortController?: AbortController): Promise<ExecaChildProcess> {
+  private async exec(
+    args: string[],
+    abortController?: AbortController,
+    waitForInit = true
+  ): Promise<ExecaChildProcess> {
+    if (waitForInit) await this.initPromise;
     return execa(this.cliPath, args, { env: this.env, input: "", signal: abortController?.signal });
```

Comment:
Positional parameters quickly get inconvenient, so it's more clear accept an `options` object here. 
This way we also avoid passing `undefined` on the `abortController` 🙂

```suggestion
    options?: { abortController?: AbortController, waitForInit?: boolean }
  ): Promise<ExecaChildProcess> {
    const { abortController, waitForInit = true } = options ?? {};
    if (waitForInit) await this.initPromise;
    return execa(this.cliPath, args, { env: this.env, input: "", signal: abortController?.signal });
```

Reply:
This is much better. Thanks!

Coming from Python most of the time I tried to call the functions with `exec(["logout", waitForInit = true)` 🤦